include/OpenColorIO/OpenColorTransforms.h

```diff
@@ -864,15 +848,9 @@ OCIO_NAMESPACE_ENTER
         TransformRcPtr & getTransform(int index);
 
         //!cpp:function::
-        int size() const;
-        //!cpp:function:: Adds a copy of transform to the group.
-        void push_back(const ConstTransformRcPtr & transform);
-        //!cpp:function:: Adds transform to the group.
-        void push_back(TransformRcPtr & transform);
-        //!cpp:function:: Clears transforms (not the FormatMetadata).
-        void clear();
```

Comment:
I like the new GroupTransform interface, but wondering if we should keep `clear()` or another way to remove transforms. I haven't had a need for it in practice myself yet, but there could be cases.

Reply:
Could also be a pop or insert method or something. Not to go overkill in making a vector interface... since it seems like your goal is to get away from it.

Reply:
The goal was to simply the interface. Most common use is to create a group transform and to add transforms to it, so there is no real need for erase, insert etc.
If a empty group transform is needed, user can use GroupTransform::Create.

Reply:
Ok. Seems fair. Users can iterate one GroupTransform and selectively add transforms to a new empty GroupTransform if variation is needed.